### ISSUE TITLE
All the projects are confusing!

### ISSUE THREAD

**Issue by boehs** [polcak/jsrestrictor#136]
So, we have https://jshelter.org and https://polcak.github.io/jsrestrictor/, both have different uis but share the chrome extension link. Many of the pages on both sites are replicas content wise, but jshelter links to https://pagure.io/JS-Shield/JS-Shield.

The chrome extension has a different logo than the crab, and JShelter and Javascript Restrictor are two different names

- This repo has the namejsrestrictor
- JShelter shares the new? crab logo, but calls the extension JShelter
- Everywhere else its called jsrestrictor
- FSF refers to it as JShelter https://www.fsf.org/news/fsf-announces-jshelter-browser-add-on-to-combat-threats-from-nonfree-javascript

People on reddit were treating the extension like its new, but in reality the repos have been around for months

So

- What connection does JShelter and JSrestrictor have?
- What is the website I should link to?
- Was it just a name change JSrestrictor to JShelter and stuff is slow to update?
- Why did FSF just announce it
- Is the crab logo new?

**Comment by TechnologyClassroom** [comment]
tl;dr The project is undergoing a rebranding currently and not all of the parts have updated yet.

The working titles were JS-Shield and JavaScript Restrictor.  The current name is JShelter which was recently decided.

The website that should be linked is https://jshelter.org/

The FSF announcement was in the works for a long time with drafts going back and forth between organizations.

The crab logo is new.
